client/stylesheets/_responsive.scss

```diff
@@ -55,6 +62,14 @@
     .searchResultsCardCondition, .searchResultsCardLocation {
         font-size: 1rem;
     }
+
+    // .profileCardImg {
+    //     height: 128px !important;
+    // }
+
+    .managerOfferView {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `managerOfferView` should be written in lowercase with hyphens